ui/src/app/apps/apps.component.html

```diff
@@ -16,7 +16,7 @@ <h1 class="page-title-actions">
 <clr-datagrid
   (clrDgRefresh)="refresh($event)"
   [clrDgLoading]="loading"
-  [(clrDgSelected)]="!grouped ? null : selected"
+  [(clrDgSelected)]="selected"
```

Comment:
~What is the impact of dropping the `!grouped` condition?~

I think this is now handled [here](https://github.com/spring-cloud/spring-cloud-dataflow-ui/pull/2028/files#diff-0266c6560b0757515b333e80136a92e0b839eea22e3998cd0232ed673f953f7dR53) correct? If so, please resolve this comment.